apps/admin/frontend/src/screens/tally_screen.tsx

```diff
@@ -121,143 +130,137 @@ export function TallyScreen(): JSX.Element | null {
+                      <TD>{getPrecinctNames(precinctIds)}</TD>
                     </tr>
-                  ) : null}
-                  <tr>
+                  )
+                )}
+                {hasManualTally ? (
+                  <tr key="manual-data">
                     <TD />
-                    <TD as="th" narrow nowrap>
-                      Total CVRs Count
+                    <TD narrow nowrap>
+                      {moment(manualTallyFirstAdded).format(TIME_FORMAT)}
                     </TD>
-                    <TD as="th" narrow data-testid="total-cvr-count">
-                      {format.count(
-                        castVoteRecordFileList.reduce(
-                          (prev, curr) => prev + curr.numCvrsImported,
-                          0
-                        ) + manualTallyTotalBallotCount
-                      )}
+                    <TD narrow>{format.count(manualTallyTotalBallotCount)}</TD>
+                    <TD narrow nowrap>
+                      Manually Entered Results
                     </TD>
-                    <TD />
-                    <TD as="th" />
+                    <TD>{getPrecinctNames(manualTallyPrecinctIds)}</TD>
                   </tr>
-                </React.Fragment>
-              ) : (
+                ) : null}
                 <tr>
-                  <TD colSpan={3}>
-                    <em>No CVR files loaded.</em>
+                  <TD />
+                  <TD as="th" narrow nowrap>
+                    Total CVRs Count
+                  </TD>
+                  <TD as="th" narrow data-testid="total-cvr-count">
+                    {format.count(
+                      castVoteRecordFileList.reduce(
+                        (prev, curr) => prev + curr.numCvrsImported,
+                        0
+                      ) + manualTallyTotalBallotCount
+                    )}
                   </TD>
+                  <TD />
+                  <TD as="th" />
                 </tr>
-              )}
-            </tbody>
-          </Table>
-          <h2>Manually Entered Results</h2>
-          <p>
-            <LinkButton
-              to={routerPaths.manualDataSummary}
-              disabled={isOfficialResults}
-            >
-              {hasManualTally
-                ? 'Edit Manually Entered Results'
-                : 'Add Manually Entered Results'}
-            </LinkButton>{' '}
-            <Button
-              disabled={!hasManualTally || isOfficialResults}
-              onPress={() => setIsConfirmingRemoveAllManualTallies(true)}
-            >
-              Remove Manually Entered Results
-            </Button>
-          </p>
-        </Prose>
+              </React.Fragment>
+            ) : (
+              <Caption>
+                <Icons.Info /> No CVR files loaded.
+              </Caption>
+            )}
+          </tbody>
+        </Table>
+        <H2>Manually Entered Results</H2>
```

Comment:
It seems a bit odd that the manually entered results section gets a heading but the CVR section doesn't. Maybe that looked different when the H1 wasn't visually separated in the old design

Reply:
Yeah, thought so too...and now that you mention it, maybe the "CVR Management" heading was always meant as a section heading instead of a page heading - maybe better to have the heading match the nav button and be "Tally" and then move "CVR Management" down to a section header for the CVR buttons/table

Reply:
Nice, that makes sense

Reply:
Do the rest of the pages have headings that match the tab labels? Might be worth a quick pass

Reply:
Yeah, all the others do, IIRC (some of them are just longer versions of the nav labels), but will check on them to be sure.

Headings updated in https://github.com/votingworks/vxsuite/pull/3632/commits/3357166decff0566dec538b06856f02097b3f5f2 - looks better now:
<img width="1735" alt="Screenshot 2023-06-26 at 16 22 10" src="https://github.com/votingworks/vxsuite/assets/264902/407a91ea-974a-48b6-ad8d-15bd849bcfe3">